x-pack/plugins/cases/public/components/recent_cases/index.test.tsx

```diff
@@ -160,8 +167,29 @@ describe('RecentCases', () => {
             username: 'elastic',
           },
         ],
+        owner: ['securitySolution'],
       },
       queryParams: { perPage: 10 },
     });
   });
+
+  it('sets all available solutions correctly', () => {
+    appMockRender = createAppMockRenderer({ owner: [] });
+    /**
+     * We set securitySolutionCases capability to not have
+     * any access to cases. This tests that we get the owners
+     * that have at least read access.
+     */
+    appMockRender.coreStart.application.capabilities = {
+      ...appMockRender.coreStart.application.capabilities,
+      securitySolutionCases: noCasesCapabilities(),
+    };
+
+    appMockRender.render(<RecentCases {...{ ...defaultProps, maxCasesToShow: 2 }} />);
+
+    expect(useGetCasesMock).toHaveBeenCalledWith({
+      filterOptions: { reporters: [], owner: ['cases'] },
```

Comment:
Should `observability` be here as well?

Reply:
The mocks here `x-pack/plugins/cases/public/common/lib/kibana/kibana_react.mock.ts` only add `generalCases` in the capabilities. So we have `generalCases` with `read_cases: true` + `securitySolutionCases` (added in the test) with `read_cases: false`.